docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,93 @@
+# Contributing
+
+
+Contributions are welcome and are greatly appreciated! Every little bit helps, and credit will always be given.
+
+You can contribute in many ways:
+
+## Types of Contributions
+
+### Report Bugs
+
+
+Report bugs by [submitting an issue](https://github.com/LibraryOfCongress/concordia/issues).
+
+If you are reporting a bug, please include:
+
+* Your operating system name and version.
+* Any details about your local setup that might be helpful in troubleshooting.
+* Detailed steps to reproduce the bug.
+
+### Submit a pull request to fix bugs
+
+Look through the GitHub issues for bugs. Anything tagged with "bug" is open to whoever wants to implement it.
+
+### Submit a pull request to implement features
+
+Look through the GitHub issues for features. Anything tagged with "feature" is open to whoever wants to implement it.
+
+### Write Documentation
+
+
+Concordia could always use more documentation. If you have worked in the tool and found any of our documentation in accurate or requires more specificity, submit an issue or a pull request. 
+
+### How to submit Feedback
+
+
+The best way to send feedback is to file an issue at https://github.com/LibraryOfCongress/concordia/issues.
+
+If you are proposing a feature:
+
+* Explain in detail how it would work.
+* Keep the scope as narrow as possible, to make it easier to implement.
+* Remember that this is a volunteer-driven project, and that contributions
+  are welcome :)
+
+Get Started!
+------------
+
+Ready to contribute? Check out our [README](https://github.com/LibraryOfCongress/concordia/blob/master/README.rst) on how to set up your machine for local development.
```

Comment:
I don't think this will ever matter but if we used "../README.rst" it should work on branches other than master or forks, which might avoid a little confusion if someone is updating the docs.